src/promptflow/promptflow/_sdk/_serving/app.py

```diff
@@ -8,27 +8,27 @@
 import logging
 from pathlib import Path
 
-from flask import Flask, jsonify, request, g
```

Comment:
Why order changed here? Maybe run `pre-commit install` in root folder to avoid checking in format changes?